92-eth-discovery.md

```diff
@@ -0,0 +1,51 @@
+## Preamble
+
+    Idea: #92-disc-v5-research
```

Comment:
Should we rename this to #92-eth-discovery so it is the same as Slack channel (IIRC)?

Reply:
On Slack it's `#92-disc-v5-research`.

Reply:
Oh right, sorry. Should probably rename this file then